Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14-alpine3.10
+
+ENV NODE_VERSION 14.9.0
+
+WORKDIR /app
+
+# Add package.json before rest of repo for caching
+COPY package.json /app/
```

Comment:
can we do this way?

```bash
COPY package.json package-lock.json /app/
```
just one line

Reply:
does not effect for that. 
p/s:  you can use git suggestion that help the author change code faster :D

Reply:
can u change it into 1 line like I suggested?